src/reynir_correct/errtokenizer.py

```diff
@@ -1851,21 +1843,21 @@ def only_suggest(token: CorrectToken, m: Sequence[BIN_Tuple]) -> bool:
                     continue
 
         # Check for completely unknown and uncorrectable words
-        # TODO STILLING - hér er ósamhengisháð leiðrétting!
         if not token.val:
             # No annotation and not able to correct:
             # mark the token as an unknown word
             # (but only as a warning if it is an uppercase word or
             # if we're within parentheses)
-            token.set_error(
-                UnknownWordError(
-                    code="001",
-                    txt="Óþekkt orð: '{0}'".format(token.txt),
-                    original=token.txt,
-                    suggest="",
-                    is_warning=token.txt[0].isupper() or bool(parenthesis_stack),
+            if token.txt[0].islower():
+                token.set_error(
+                    UnknownWordError(
+                        code="001",
+                        txt="Óþekkt orð: '{0}'".format(token.txt),
+                        original=token.txt,
+                        suggest="",
+                        is_warning=token.txt[0].isupper() or bool(parenthesis_stack),
```

Comment:
This condition can never be True (token.txt[0].islower() is checked in the if statement above)

Reply:
Done